src/pages/professors.tsx

```diff
@@ -1,30 +1,28 @@
 import {
     Box,
-    Table,
-    TableContainer,
-    Tbody,
-    Td,
-    Th,
-    Thead,
-    Tr,
 } from "@chakra-ui/react";
 import { ReactElement, useState } from "react";
 
-import { CompleteStatusBadge } from "../components/CompleteStatusBadge";
 import AdminLayout from "../components/Layout/AdminLayout";
+import ProfessorsTable from "../components/Professors/ProfessorsTable";
 import SampleSidesheet from "../components/Sample/SampleSidesheet";
 
 const Professors = ({ professors }) => {
     const [open, setOpen] = useState(false);
 
+    const openDetails = (prof) => {
+        // can use prof values here (from backend)
```

Comment:
looks good, might benefit from a custom hook in the future
will probably just need the prof id for the query